wallet/test/test_utils.h

```diff
@@ -21,4 +21,7 @@
 
 #define CHECK(cond) CHECK_MSG(cond,"test condition failed");
 
+#include "lightningd/lightningd.h"
+static const struct config test_config;
```

Comment:
Not sure what `static` is used for in this case. It'll work without it as well :-)

Reply:
addressed a22b11093bee9d7a9e0aec0a4703c8f9db116d66

The `options.c` testnet and mainnet configs are also static, maybe because of that, removed it anyway.